docs/concepts/copilot/copilot-ai-capabilities.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Data Copilot AI capabilities
+id: copilot-ai-capabilities
+description: The AI assistant capabilities for data Pipelines and models
+sidebar_position: 9
+tags:
+  - concepts
+  - copilot
+  - generativeai
+  - capabilities
+---
+
+Prophecy Data Copilot provides suggestions from an AI model as you develop your data Pipelines and Models. You can view and incorporate suggestions directly within the Prophecy visual editor and code editor. Data Copilot makes suggestions for your entire Pipeline, for a single Gem (transformation), and even for individual expressions within each Gem.
+
+## Supported AI capabilities by engine
+
+Data Copilot supports the following capabilites for the Spark and SQL engines:
+
+| AI Capability                 | Spark                    | SQL                      |
+| ----------------------------- | ------------------------ | ------------------------ |
+| Start a new Pipeline          | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
```

Comment:
Consider mentioning that pipelines are called models for SQL:  

Start a new pipeline(Spark) or model(SQL)               ✔️   ✔️ 
Modify an existing pipeline(Spark) or model(SQL)    ❌    ✔️

Reply:
Done.